avaframe/com1DFA/com1DFA.py

```diff
@@ -1608,11 +1661,19 @@ def initializeResistance(cfg, dem, simTypeActual, resLine, reportAreaInfo, thres
         simulation area information dictionary completed with entrainment area info
     """
     cRes = cfg.getfloat("cRes")
+    DetBool = cfg.getboolean("DetBool")
+    DetAndRes = cfg.getboolean("DetAndRes")
```

Comment:
If we want to treat this separate and have separate functions, this logic should be treated outside this function. I.e. the algorithm should look like (for different functions):
- check if res
- then check if det on or off
- call the needed initialize functions.

OR: for only one function:

- call initialize function
- decide inside whether res and det are needed

Question is: (also @awirb ) do we want to treat it separately?

Reply:
Consensus is:
no extra runtype, detrainment is an option flag of the resistance